tests/test_product/product_test.go

```diff
@@ -132,3 +132,123 @@ func TestProductCreate(t *testing.T) {
 	}
 
 }
+
+// func TestProductGet(t *testing.T) {
+// 	logger := tst.Setup()
+// 	gin.SetMode(gin.TestMode)
+//
+// 	validatorRef := validator.New()
+// 	db := storage.Connection()
+// 	requestURI := url.URL{Path: "/api/v1/products"}
+// 	currUUID := utility.GenerateUUID()
+// 	userSignUpData := models.CreateUserRequestModel{
+// 		Email:       fmt.Sprintf("johncarpenter%v@qa.team", currUUID),
+// 		PhoneNumber: fmt.Sprintf("+234%v", utility.GetRandomNumbersInRange(7000000000, 9099999999)),
+// 		FirstName:   "test",
+// 		LastName:    "user",
+// 		Password:    "password",
+// 		UserName:    fmt.Sprintf("test_username%v", currUUID),
+// 	}
+// 	loginData := models.LoginRequestModel{
+// 		Email:    userSignUpData.Email,
+// 		Password: userSignUpData.Password,
+// 	}
+//
+// 	auth := auth.Controller{Db: db, Validator: validatorRef, Logger: logger}
+// 	r := gin.Default()
+// 	tst.SignupUser(t, r, auth, userSignUpData, false)
```

Comment:
remove this comments too

Reply:
they are still needed but i don't want that to slow the progress of other tasks 
i will write them when the main thing is done